test/cypress/dashboards/integration/EditMode/component-editing.cy.js

```diff
@@ -0,0 +1,40 @@
+describe('Path to arrays in editable options.', () => {
+
+    function openTextArea() {
+        cy.get('.highcharts-dashboards-component').first().click();
+        cy.get('.highcharts-dashboards-edit-toolbar-cell .highcharts-dashboards-edit-toolbar-item').eq(1).click();
+
+        return cy.get('.highcharts-dashboards-edit-accordion').eq(1).click().get('textarea');
+    }
+
+    function cancelChanges() {
```

Comment:
`cancelChanges` and `submitChanges` could be declared as cypress commands- we might use them in the future